### PR TITLE
Add existing spotbugs violations as baseline

### PR DESCRIPTION
**Pull request by iamgusain**
Adding existing spotbugs violations as [baseline](https://spotbugs-gradle-plugin.netlify.app/com/github/spotbugs/snom/spotbugsextension#baselineFile). So that we can catch new violations.
I will create a separate work item to fix these violations and then get rid of the baseline file
Workitem: https://identitydivision.visualstudio.com/Engineering/_backlogs/backlog/Auth%20Client%20-%20Android/Features/?workitem=1381102

### CODE REVIEW

**Comment by rpdome** [comment]
Could you add some more details on how this file works / was generated?

as in, how am I supposed to review this? :sweater

**Comment by iamgusain** [comment]
> Could you add some more details on how this file works / was generated?
> 
> as in, how am I supposed to review this? 😅

The baseline file, that contains existing spotbugs rule violations that we have in the repo. Such that when we run spotbugs task it will not report these issues as violations, and will only report new ones. 
The file is auto generated by running spotbugs on existing code, 
More info: https://spotbugs-gradle-plugin.netlify.app/com/github/spotbugs/snom/spotbugsextension#baselineFile

File: config/spotbugs/baseline.xml [hunk 1]
```diff
@@ -0,0 +1,1200 @@
...
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\UserInfo.class</Jar>
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\WebFingerMetadata.class</Jar>
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\WebFingerMetadataRequestor.class</Jar>
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\WebFingerMetadataRequestParameters.class</Jar>
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\WebviewHelper$PreKeyAuthInfo.class</Jar>
+    <Jar>C:\temp\android_auth\adal\intermediates\javac\localDebug\classes\com\microsoft\aad\adal\WebviewHelper.class</Jar>
+    <AuxClasspathEntry>C:\temp\android_auth\adal\intermediates\compile_r_class_jar\localDebug\R.jar</AuxClasspathEntry>
+    <AuxClasspathEntry>C:\Users\amgusain\.gradle\caches\transforms-2\files-2.1\b6fbbdadfcff642127829062bce22562\appcompat-1.0.2-api.jar</AuxClasspathEntry>
```
Comment (iamgusain) [inline config/spotbugs/baseline.xml:178]: Need to remove machine specific paths.

**Review by tanmaymanolkar1 — APPROVED** [review]
LGTM

File: config/spotbugs/baseline.xml (hunk 1 above)
Comment (iamgusain) [reply config/spotbugs/baseline.xml]: Done

**Pull request merged**